recipes/opensim/build.sh

```diff
@@ -0,0 +1,59 @@
+#!/bin/bash
+
+# The build instructions for OpenSim 3.3 can be found at:
+# http://simtk-confluence.stanford.edu:8080/display/OpenSim/Building+OpenSim+from+Source
+
+# The following are the commands to manually build Opensim inside a conda
+# environment.
+#mkdir build
+#cd build
+#cmake .. \
+	#-DCMAKE_INSTALL_PREFIX="$CONDA_PREFIX" \
+	#-DCMAKE_BUILD_TYPE=Release \
+	#-DBUILD_API_EXAMPLES=On \
+	#-DBUILD_TESTING=On \
+	#-DBUILD_JAVA_WRAPPING=Off \
+	#-DBUILD_PYTHON_WRAPPING=Off \
+	#-DOPENSIM_STANDARD_11=On \
+	#-DSIMBODY_HOME="$CONDA_PREFIX" \
+	#-DBUILD_USING_OTHER_LAPACK="$CONDA_PREFIX/lib/libopenblas.so"
+
+# FIXME: This is a hack to make sure the environment is activated. The reason
+# this is required is due to the conda-build issue mentioned below:
+# https://github.com/conda/conda-build/issues/910
+#source activate "${CONDA_DEFAULT_ENV}"
+
+mkdir build
+cd build
+if [[ "$OSTYPE" == "linux-gnu" ]]; then
+	SHARED_EXT=so
+elif [[ "$OSTYPE" == "darwin"* ]]; then
+	SHARED_EXT=dylib
+fi
+cmake ../OpenSim33-source \
+	-DCMAKE_INSTALL_PREFIX="$PREFIX" \
+	-DCMAKE_BUILD_TYPE=Release \
+	-DBUILD_API_EXAMPLES=On \
+	-DBUILD_TESTING=On \
+	-DBUILD_JAVA_WRAPPING=Off \
+	-DBUILD_PYTHON_WRAPPING=Off \
```

Comment:
Because this is only the C++ lib and headers. The python bindings will be in a separate recipe.